craco.config.js

```diff
@@ -0,0 +1,18 @@
+const CracoLessPlugin = require('craco-less');
+
+module.exports = {
+  plugins: [
+    {
+      plugin: CracoLessPlugin,
+      options: {
+        lessLoaderOptions: {
+          lessOptions: {
+            //This is where we override LESS global variables
```

Comment:
❓ Are these variables indexable, like, in IntelliJ?

Reply:
Not sure what you mean? We can import/require and format into whatever we want though!